### DISCUSSION TITLE
Get number of occurrences of a word in a document

### DISCUSSION THREAD

**Discussion by JfuTRT** [microsoft-search/pnp-modern-search#1864] [Q&A]
Hello team!

I'm building a SharePoint Online Search using pnp modern search v4 and I'm trying to add the number of occurrences that a word appears in a document, but I'm not able to do. 

Can anybody help me? 

Thank you in advance!

**Comment by wobba** [comment]
There's no way you can do that oob as that information is not stored in the search index.
